examples/vision/densenet_cspnet_variants.py

```diff
@@ -0,0 +1,396 @@
+"""
+Title: DenseNet and CSPNet
+Author: Varchita Lalwani
+Date created: 27 June 2021
+Last modified: 27 June 2021
+Description: Training DenseNet, CSPNet and variants on Cifar10 Dataset.
+"""
+"""shell
+densenet_cspnet_variants.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1ejnjgRzfo60FWXMp8_HyzQJ_5kBLIdwd
+
+##DenseNet
```

Comment:
The example should explain the point of these models -- why should a reader be interested in them? How should they use them? How do they compare to alternatives?

The example will also need to be polished much further, as it currently has many issues. Please check out README instructions for how to develop and preview code examples.